api/v1/mongodbcommunity_types.go

```diff
@@ -373,6 +379,15 @@ func (m MongoDBUser) GetScramCredentialsSecretName() string {
 	return fmt.Sprintf("%s-%s", m.ScramCredentialsSecretName, "scram-credentials")
 }
 
+// GetConnectionStringSecretName gets the connection string secret name provided by the user or generated
+// from the SCRAM user configuration.
+func (m MongoDBUser) GetConnectionStringSecretName(resourceName string) string {
```

Comment:
Isn't `resourceName` available in the receiving `MongoDBUser`? Can you skip the `resourceName` parameter?

Reply:
It's not available in the `MongoDBUser` indeed.